spond/spond.py

```diff
@@ -381,3 +381,29 @@ async def update_event(self, uid, updates: dict):
         ) as r:
             self.events_update = await r.json()
             return self.events
+
+async def get_post(self, uid, prevuid, date) -> dict:
+        """
+        Get a group by unique ID.
+        Subject to authenticated user's access.
+
+        Parameters
+        ----------
+        uid : str
+            UID of the group.
+        prevuid : str
+            Different UID used on the webpage, optional to use when calling this function.
```

Comment:
How would someone actually use `prevuid` - can you give an example? If it's optional, it should be coded as such.